src/partials/exercises.html

```diff
@@ -22,10 +22,8 @@ <h2>Exercises</h2>
             <p class="rating"></p>
             <button class="start">Start</button>
             <svg class="arrow" width="14" height="14">
-              <use
-                href="/src/img/sprite/sprite.svg#icon-exercises-
-            arrow"
-              ></use>
+              <use href="/src/img/sprite/sprite.svg#icon-exercises-
```

Comment:
Path should be with dot. example: `<use href="./img/sprite/sprite.svg#icon-exercises-arrow"></use>`
Chang all paths.